ovos_utils/gui.py

```diff
@@ -743,7 +760,13 @@ def send_event(self, event_name: str,
                                "params": params}))
 
     def _pages2uri(self, page_names: List[str]) -> List[str]:
```

Comment:
deprecated decorator?

Reply:
It's an internal method, so I think it's fine to deprecate without warning once we deprecate everything consuming `message.data['page']`